media/css/mozorg/contribute/taskview.less

```diff
@@ -0,0 +1,444 @@
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        padding-left: 10px;
+        padding-right: 10px;
+    }
+
+    .task-heading {
+        margin: (@baseLine * 3) auto;
+        width: 100%;
+        text-align: center;
+
+        @media only screen and (max-width: @breakTablet) {
+            margin: 10px auto;
+        }
+
+        p {
+            margin: 0 auto;
+            max-width: 80%;
+            .open-sans-light;
+            .font-size(21px);
+            font-style: italic;
+
+            @media only screen and (max-width: @breakMobileLandscape) {
+                .font-size(18px);
+                width: 90%;
+                max-width: inherit;
+            }
+        }
+    }
+
+    .page-title {
+        margin: 0 auto @baseLine;
+        max-width: 580px;
+        .open-sans-light;
+        .font-size(48px);
+        line-height: 1.3;
+
+        @media only screen and (max-width: @breakTablet) {
+            max-width: 500px;
+            .font-size(40px);
+        }
+
+        @media only screen and (max-width: @breakMobileLandscape) {
+            .font-size(32px);
+        }
+    }
+}
+
+.task-steps {
+    margin: (@baseLine * 2) auto 0;
+    list-style: none;
+    max-width: 860px;
+
+    @media only screen and (max-width: @breakDesktop) {
+        max-width: 720px;
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        max-width: 600px;
+    }
+
+    li {
+        margin: 0 0 (@baseLine * 2);
+        width: 80%;
+
+        &:before {
+            display: inline-block;
+            border: 2px solid @foregroundColor;
+            border-radius: 50%;
+            width: 50px;
+            height: 50px;
+            .font-size(24px);
+            font-weight: bold;
+            line-height: 2;
+            text-align: center;
+
+            @media only screen and (max-width: @breakTablet) {
+                width: 30px;
+                height: 30px;
+                .font-size(18px);
+                line-height: 1.6;
+            }
+        }
+    }
+
+    header {
+        margin-top: -50px;
+        margin-left: 70px;
+        width: 100%;
+
+        @media only screen and (max-width: @breakTablet) {
+            margin-top: -32px;
+            margin-left: 45px;
+        }
+    }
+
+    h3 {
+        .open-sans;
+        .font-size(32px);
+        font-weight: bold;
+        text-transform: uppercase;
+        line-height: 1.3;
+
+        @media only screen and (max-width: @breakTablet) {
+            .font-size(24px);
+        }
+
+        @media only screen and (max-width: @breakMobileLandscape) {
+            .font-size(19px);
+        }
+    }
+
+    .step-content {
+        display: inline-block;;
+        margin-left: 70px;
+        width: 100%;
+
+        @media only screen and (max-width: @breakTablet) {
+            margin-left: 45px;
+        }
+    }
+
+    .one:before {
+        content: '1';
+    }
+
+    .two:before {
+        content: '2';
```

Comment:
Just a note that if/when we do localize, will have to figure out a different way to display these numbers.